products-service/serverless.ts

```diff
@@ -1,4 +1,5 @@
 import { Serverless } from 'serverless/aws';
+import { PG_HOST, PG_PORT, PG_DATABASE, PG_USERNAME, PG_PASSWORD } from './config';
```

Comment:
[proposal]
Consider also using https://www.serverless.com/plugins/serverless-dotenv-plugin to setup auto mapping for env variables from .env file